docs/source/gbd2020_models/risk_attributable_causes/wasting/index.rst

```diff
@@ -571,19 +571,19 @@ that follows we will detail how to calculate all the variables used
      - Daily probability of incidence into cat 3 from cat 4
      - System of equations
    * - r2
-     - (1-sam_tx_coverage)*(1/time_to_sam_ux_recovery)
+     - 1 - e^((1-sam_tx_coverage)*(1/time_to_sam_ux_recovery))
```

Comment:
Should there be a negative sign in the exponent?
I.e. `1 - e^(-(1-sam_tx_coverage)*(1/time_to_sam_ux_recovery))`

Reply:
Oops, yes! Just updated, thanks.